### PR TITLE
Implement translate button on quick menu

### PR DESCRIPTION
**Pull request by jdragonbae**
This patch implements translate button on quick menu. Since there is a
limit on the number of quick menu buttons, we decided to replace add to
home with this.

Refs: #1049

### CODE REVIEW

File: upstream/tb_components_translate_core_browser_translate_manager.cc.patch [hunk 1]
```diff
@@ -0,0 +1,31 @@
+diff --git tb/components/translate/core/browser/translate_manager.cc tb_diff/components/translate/core/browser/translate_manager.cc
+--- tb/components/translate/core/browser/translate_manager.cc
++++ tb_diff/components/translate/core/browser/translate_manager.cc
+@@ -265,6 +265,7 @@ bool TranslateManager::CanManuallyTranslate(bool menuLogging) {
+   // Translation of unknown source language pages is supported on desktop
+   // platforms, but not mobile.
+ #if defined(OS_ANDROID) || defined(OS_IOS)
++#if !defined(ENABLE_TRIPLE_BANANA)
```
Comment (romandev) [inline upstream/tb_components_translate_core_browser_translate_manager.cc.patch:8]: So, if we enable this code by force, the mobile web page is actually translated? or not?

**Review by romandev — COMMENTED** [review]
lgtm

File: upstream/tb_components_translate_core_browser_translate_manager.cc.patch (hunk 1 above)
Comment (jdragonbae) [reply upstream/tb_components_translate_core_browser_translate_manager.cc.patch:8]: The mobile web page is actually translated in spite of `unknown language`.

**Review by romandev — APPROVED** [review]
lgtm

**Pull request merged**